libs/schema-profunctor/src/Data/Schema.hs

```diff
@@ -296,6 +296,7 @@ instance HasOpt doc => FieldFunctor doc Maybe where
 
 -- | A schema for a one-field JSON object.
 field ::
+  forall doc' doc a b.
```

Comment:
Why are explicit foralls necessary now?

Reply:
We need to scope the type variables to be able to use type applications. I'm not sure what change in GHC caused this. Perhaps it had something to do with the Deepsubsumption change